Knowledge/react-app/src/App.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import Header from './components/Header'
+import HeroImage from './components/HeroImage';
+import Footer from './components/Footer';
+import './App.scss';
+
+
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
```

Comment:
You can change the css name of "App" to 'app" so to have a more consistency with BEM naming conventions